cli/conf.py

```diff
@@ -43,14 +101,8 @@ def load_config_file():
         for option, value in DEFAULT_CONF_DIRS.iteritems():
             _config.set('defaults', option, os.path.join(project_dir, value))
 
-    # Validates settings dir exists
-    settings_dir = _config.get('defaults', 'settings')
-    if not os.path.exists(settings_dir):
-        raise exceptions.IRFileNotFoundException(
-            settings_dir,
-            "Settings directory doesn't exist: ")
-
     return _config
 
 
-config = load_config_file()
+config = ConfigWrapper(load_config_file())
```

Comment:
What do you think about moving the 'load_config_file' function into the 'ConfigWrapper' class?